src/Publish/Targets/ImportPublishProfile.targets

```diff
@@ -0,0 +1,34 @@
+<!--
+***********************************************************************************************
+ImportPublishProfile.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!--
+  ***********************************************************************************************
+  Import the PublishProfiles
+  ***********************************************************************************************
+ -->
+
+  <PropertyGroup>
+    <_PublishProfileImported>true</_PublishProfileImported>
+    <_PublishProfileRootFolder Condition="'$(_PublishProfileRootFolder)' == ''">$(MSBuildProjectDirectory)\$(AppDesignerFolder)\PublishProfiles\</_PublishProfileRootFolder>
+    <PublishProfile Condition="'$(PublishProfile)' ==''">FileSystem</PublishProfile>
+    <PublishProfileName Condition="'$(PublishProfileName)' == ''">$([System.IO.Path]::GetFileNameWithoutExtension($(PublishProfile)))</PublishProfileName>
+    <PublishProfileFullPath Condition="'$(PublishProfileFullPath)' == ''">$(_PublishProfileRootFolder)$(PublishProfileName).pubxml</PublishProfileFullPath>
+    <PublishProfileFullPath Condition="!Exists('$(PublishProfileFullPath)')">$(MSBuildThisFileDirectory)PublishProfiles\$(PublishProfileName).pubxml</PublishProfileFullPath>
```

Comment:
This is wrong. The default profiles will also have to move. 

$(MSBuildThisFileDirectory)PublishProfiles